src/diffusers/pipelines/marigold/pipeline_marigold_depth.py

```diff
@@ -0,0 +1,661 @@
+                Preferred format of the output's `prediction` and the optional `uncertainty` fields. The accepted
+                values are: `"pil'` (`PIL.Image.Image`), `"np"` (numpy array) or `"pt"` (torch tensor).
+            output_visualization (`bool`, *optional*, defaults to `True`):
+                When enabled, the output's `visualization` field contains a PIL.Image that can be used for visual
+                quality inspection.
+            output_visualization_kwargs (`dict`, *optional*, defaults to `None`):
+                Extra dictionary with arguments for precise visualization control. The following options are available:
+                - color_map (`str`, *optional*, defaults to `"Spectral"`): Color map used to convert a single-channel
+                  depth prediction into colored representation.
+                - vis_min (`float`, *optional*, defaults to `0.0`): Minimum value of the visualized depth range.
+                - vis_max (`float`, *optional*, defaults to `1.0`): Maximum value of the visualized depth range.
+            output_uncertainty (`bool`, *optional*, defaults to `True`):
+                When enabled, the output's `uncertainty` field contains the predictive uncertainty map, provided that
+                the `ensemble_size` argument is set to a value above 2.
+            output_latent (`bool`, *optional*, defaults to `False`):
+                When enabled, the output's `latent` field contains the latent codes corresponding to the predictions
+                within the ensemble. These codes can be saved, modified, and used for subsequent calls with the
+                `latents` argument.
+
+        Examples:
+
+        Returns:
+            `MarigoldDepthOutput`: Output class instance for Marigold monocular depth prediction pipeline.
+        """
+
+        # 0. Resolving variables
+        device = self._execution_device
+        dtype = self.dtype
+
+        num_images = 1
+        if (isinstance(image, np.ndarray) or torch.is_tensor(image)) and image.ndim == 4:
+            num_images = image.shape[0]
+
+        if num_inference_steps is None:
+            num_inference_steps = self.default_denoising_steps
+        if processing_resolution is None:
+            processing_resolution = self.default_processing_resolution
+
+        # 1. Checking inputs
+        self.check_inputs(
+            image,
+            num_inference_steps,
+            ensemble_size,
+            processing_resolution,
+            resample_method_input,
+            resample_method_output,
+            batch_size,
+            ensembling_kwargs,
+            latents,
+            generator,
+            output_prediction_format,
+            output_visualization_kwargs,
+        )
+
+        # 2. Prepare empty text conditioning. Model invocation: self.tokenizer, self.text_encoder
+        if self.empty_text_embedding is None:
+            self.encode_empty_text()
```

Comment:
Should this rather be made a pipeline call argument? No strong opinions here.

Reply:
If we do that, someone will try to set it, and anything other than an empty string will break the current models. What we could do is adding a boolean flag `is_text_conditional` to the config, and set the default value of prompt to "". Let me know